distributions/otelcol/manifest.yaml

```diff
@@ -2,45 +2,45 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-releases/core
   name: otelcol
   description: OpenTelemetry Collector
-  version: 0.77.0
+  version: 0.78.0
   output_path: ./_build
-  otelcol_version: 0.77.0
```

Comment:
Should this be 0.78.2? I thought this was the core components.

Reply:
Done.